src/dialect/duckdb.rs

```diff
@@ -94,4 +94,12 @@ impl Dialect for DuckDbDialect {
     fn supports_order_by_all(&self) -> bool {
         true
     }
+
+    fn supports_not_null(&self) -> bool {
+        true
+    }
+
+    fn supports_notnull(&self) -> bool {
+        true
+    }
```

Comment:
I also considered makng these one function:

```rust
fn supports_not_null(&self, with_space: bool) -> bool { 
  false
}
```

Let me know if you prefer that.